awx/ui/client/features/templates/templates.strings.js

```diff
@@ -105,7 +105,7 @@ function TemplatesStrings (BaseString) {
         INVENTORY_SYNC: t.s('Inventory Sync'),
         WORKFLOW: t.s('Workflow'),
         WARNING: t.s('Warning'),
-        TOTAL_TEMPLATES: t.s('TOTAL TEMPLATES'),
```

Comment:
double-checked that we weren't using the removed string elsewhere and we weren't 👍